zulipterminal/ui_tools/boxes.py

```diff
@@ -101,6 +103,16 @@ def autocomplete_streams(self, text: str, state: int) -> Optional[str]:
         except IndexError:
             return None
 
+    def autocomplete_mentions(self, text: str, state: int) -> Optional[str]:
```

Comment:
can we add support `@"aman agrawal"` then we press `Ctrl f` to autocomplete it as `@**Aman Agrawal** `?

Reply:
Sounds great!